reference/6/Microsoft.PowerShell.Core/About/about_CommonParameters.md

```diff
@@ -151,126 +145,123 @@ $a
 ```
 
 You can use this parameter to create a variable that contains only error
-messages from specific commands. The $Error automatic variable contains error
-messages from all the commands in the session. You can use array notation,
-such as $a[0] or $error[1,2] to refer to specific errors stored in the
-variables.
-
--InformationAction [:{SilentlyContinue | Stop | Continue | Inquire | Ignore |
-Suspend}]
+messages from specific commands. The `$Error` automatic variable contains error
+messages from all the commands in the session. You can use array notation, such
+as `$a[0]` or `$error[1,2]` to refer to specific errors stored in the variables.
 
-Alias: ia
+#### InformationAction
 
-Introduced in PowerShell 5.0. Within the command or script in which it is
-used, the InformationAction common parameter overrides the value of the
-$InformationPreference preference variable, which by default is set to
-SilentlyContinue. When you use Write-Information in a script with
--InformationAction, Write-Information values are shown depending on the value
-of the -InformationAction parameter. For more information about
-$InformationPreference, see about_Preference_Variables.
+The alias for `-InformationAction` is `-ia`.
 
-Valid values:
+Introduced in PowerShell 5.0. Within the command or script in which it's used,
+the `-InformationAction` common parameter overrides the value of the
+`$InformationPreference` preference variable, which by default is set to
+`SilentlyContinue`. When you use `Write-Information` in a script with
+`-InformationAction`, `Write-Information` values are shown depending on the
+value of the `-InformationAction` parameter. For more information about
+`$InformationPreference`, see [about_Preference_Variables](./about_Preference_Variables.md).
 
-- Stop: Stops a command or script at an occurrence of the Write-Information
-  command.
+`-InformationAction:Stop` stops a command or script at an occurrence of the
+`Write-Information` command.
 
-- Ignore: Suppresses the informational message and continues running the
-  command. Unlike SilentlyContinue, Ignore completely "forgets" the
-  informational message; it does not add the informational message to the
-  information stream.
+`-InformationAction:Ignore` suppresses the informational message and continues
```

Comment:
`-InformationAction:Ignore` is correct markup because you are showing the usage with the value. When talking about the value, like in the next sentence, the values should be bold:

> Unlike **SilentlyContinue**, **Ignore** completely forgets the